pkg/visor/visorconfig/services.go

```diff
@@ -14,8 +14,8 @@ import (
 // Fetch fetches the service URLs & ip:ports from the config service endpoint
 func Fetch(mLog *logging.MasterLogger, serviceConfURL string, stdout bool) (services *Services) {
 
-	urlstr := []string{"http://", serviceConfURL}
-	serviceConf := strings.Join(urlstr, "")
+	urlStr := []string{"http://", serviceConfURL}
```

Comment:
Why not just use `+` operator or simple `Sprint`?

Reply:
done